### DISCUSSION TITLE
[Announcement] PaddleOCR PMC Officially Established

### DISCUSSION THREAD

**Discussion by jzhang533** [PaddlePaddle/PaddleOCR#13019] [General]
经过近期在社区的讨论： #11859 ，和线下的社区交流，我们在[社区会议](https://github.com/PaddlePaddle/community/blob/master/pposdwg/2024-Q2-meeting-minutes.md)上，正式成立了 PaddleOCR PMC(Project Management Committee) 。

## 初始 PMC 成员
- @GreatV (PMC Chair)
- @tink2123 (PMC Chair)
- @Topdu
- @SWHL
- @Liyulingyue
- @Sunting78 

## 新增PMC成员
- @dyning 

PMC 的职责，被简要地概括为：**照看项目、持续研发、发展社区**。PMC 的运作将会采用开源社区的最佳实践，尤其，我们会借鉴[Apache软件基金会](https://www.apache.org/dev/pmc.html)的最佳实践。

We are pleased to announce the formation of the PaddleOCR PMC(Project Management Committee). This committee will take responsibility for technical decision-making and the ongoing maintenance of the PaddleOCR project.

Initial PMC members: @GreatV (PMC Chair) @tink2123 (PMC Chair) @Topdu @SWHL @Liyulingyue @Sunting78 .

@jzhang533 on behalf of [PaddlePaddle Community](https://github.com/PaddlePaddle/community)

**Comment by miradel51** [comment]
The PaddleOCR project supports the two format datasets "LMDBDataSet" and "SimpleDataSet" for training data. We modified the config file "./configs/rec/rec_mv3_none_bilstm_ctc.yml" by revising the dataset name with "SimpleDataSet", and also provided the corresponding values of the parameters "data_dir" and "label_file_list".  More precisely, the "data_dir" includes a certain amount of images in "*.jpg" format, while the "label_file_list" directs to the "*.txt " file includes per line with each image absolute location "\t" image label. However, as we run the command "python  tools/train.py -c configs/rec/rec_mv3_none_bilstm_ctc.yml", it reports the error with "No Images in train dataset, please ensure\n"
            + "\t1. The images num in the train label_file_list should be larger than or equal with batch size.\n"
            + "\t2. The annotation file and path in the configuration file are provided normally."
![image](https://github.com/user-attachments/assets/43a40cc8-1993-4fb7-8786-3257c8320dd3)
